recipe/meta.yaml

```diff
@@ -1,47 +1,227 @@
 {% set name = "google-cloud-aiplatform" %}
-{% set version = "1.21.0" %}
+{% set version = "1.66.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/google-cloud-aiplatform-{{ version }}.tar.gz
-  sha256: 3cd8fdc1b38e2a21552cbdacb338c4779a27c222bb1ea2481eb56b4fa844917b
+  sha256: e95a2b45087e0e2cbfed9ffaf3f10aa8ef3fa2e5642f067246181b5c7b9b2b04
 
 build:
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  # skip s390x: many dependencies missing
+  skip: true  # [py<38 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  entry_points:
+    - tb-gcp-uploader=google.cloud.aiplatform.tensorboard.uploader_main:run_main
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - google-api-core-grpc >=1.32.0,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*
-    - proto-plus >=1.22.0,<2.0.0dev
-    - protobuf >=3.19.5,<5.0.0dev,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
-    - packaging >=14.3,<22.0.0dev
+    - google-api-core >=1.34.1,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*
+    - google-api-core-grpc
+    - proto-plus >=1.22.3,<2.0.0dev
+    - protobuf >=3.20.2,<6.0.0dev,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
+    - packaging >=14.3
+    - google-auth >=2.14.1,<3.0.0dev
     - google-cloud-storage >=1.32.0,<3.0.0dev
-    - google-cloud-bigquery-core >=1.15.0,<4.0.0dev
+    - google-cloud-bigquery-core >=1.15.0,<4.0.0dev,!=3.20.0
     - google-cloud-resource-manager >=1.3.3,<3.0.0dev
-    - shapely <2.0.0
+    - shapely <3.0.0dev
+    - docstring_parser <1
+    - pydantic <3
+  run_constrained:
+    # There are a lot of extras for this package and they often use
+    # the same packages with different pinnings. Here follow an approximation
+    # with some of those commented out, and the more restrictive ones
+    # enabled. To allow an easier update of these constraints, I leave
+    # them there and commented out.
+    #
+    # profiler
+    - tensorboard-plugin-profile >=2.4.0,<3.0.0dev
```

Comment:
I'm getting a percy error here:

```
ERROR:root:Render issue $HOME/src/AnacondaRecipes/aggregate/google-cloud-aiplatform-feedstock/recipe/meta.yaml : while parsing a block collection
  in "<unicode string>", line 48, column 5
```

which I can't quite see...

Reply:
ah thanks, I will wait for other comments to update this, as tests take long to complete